TeknoParrotUi.Common/GameProfiles/GrooveCoaster2.xml

```diff
@@ -1,14 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <GameProfile xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
-  <GameName>Groove Coaster 2: Heavenly Festival</GameName>
+  <GameName>Rhythmvaders 2</GameName>
```

Comment:
official name is Groove Coaster 2: Heavenly Festival (and if you check other games of this series, each of them is groove coaster, even the english steam version)

Reply:
Yeah but the only known dump of Groove Coaster 2: Heavenly Festival is Rhythmvaders 2. It seems it's an japenglish export version to be played in areas outside Japan.
http://rhythmvaders.com/

Reply:
And like this => Groove Coaster 2: Heavenly Festival (Rhythmvaders 2)

Reply:
If you boot the game japaneae, it says Groove Coaster 2

Reply:
OK, I'll revert it back.
Can we consider to have additionnal lines in the description fils (such as versions and alternative title)?

Reply:
> versions

Do you have any examples of games with more than one public dump available that *aren't* supported by TeknoParrot? 

This seems like a nice idea, but it would probably be pointless if only a couple of games would actually have the information filled in.

> alternative title

Probably unnecessary. If you can find more, then maybe?
In this specific case, "Rhythmvaders" has 3000 results on Google, while "Groove Coaster" has 326000.

Reply:
`versions`
I have only in mind Chaos Code New Sign of Catastrophe (1.03.00) and Dariusburst (1.05).
`Alternate title` Groove Coaster 2: Heavenly Festival > Rhythmvaders 2 / Chaos Breaker > Dark Awake: The King Has No Name / Samurai Spirits Sen > Samurai Showdown Sen / Shikigami no Shiro III > Castle of Shikigami III and so on... I was thinking about this only for this purpose but I'll use wiki for both.

Reply:
Would a UI option for alternative titles be okay with you? I don't know if having "<original> / <alternative>" would look nice.